docs: update Fivetran page for new API-key connection flow (#2313)
elementary-data/elementary: docs/cloud/integrations/pipeline/fivetran.mdx

badge: "Context Engine"
---

Elementary connects to your Fivetran account to automatically create incidents when connector syncs fail.

> When a sync fails, Elementary evaluates the outcome and opens or updates an incident accordingly.

Author: ```suggestion
When a sync ends, Elementary evaluates the outcome and opens or updates an incident accordingly.
```

<Warning>
  **Connector filtering is not yet supported.** Incidents will be created for all connectors in the monitored destination groups. Filtering by specific connectors will be added in a future release.
</Warning>

## How it works

1. You connect your Fivetran account to Elementary with a Fivetran API key and select which destination groups to monitor.

> 2. Elementary sets up sync monitoring for the selected groups automatically — no manual configuration in Fivetran is required.

Author: ```suggestion
2. Elementary subscribes to Fivetran's sync_end events for the selected groups automatically - no manual configuration in Fivetran is required.
```

3. Each time a connector sync completes, Elementary evaluates the outcome and creates or updates an incident when the sync fails.
4. The incident appears in Elementary's Incidents view, where you can investigate, comment, and resolve it.

## Enabling Fivetran

<Info>
  You need a Fivetran API key with permission to read connectors and manage webhooks, and Elementary permissions to edit integrations.
</Info>

<Steps>
  <Step title="Create a Fivetran API key">

>     In Fivetran, go to your account settings and generate an API key. Copy the **API key** and **API secret** — you'll need both in Elementary.

Author: ```suggestion
    In Fivetran, go to your account settings and generate an API key. Copy the **API key** and **API secret** .
```

    See Fivetran's [API key documentation](https://fivetran.com/docs/rest-api/getting-started) for details.
  </Step>
  <Step title="Open the Fivetran integration in Elementary">
    In Elementary, go to **Settings → Environments** and select the environment you want to connect. Under **Integrations**, click **Connect Fivetran**.
  </Step>
  <Step title="Enter your Fivetran credentials">
    Paste the **API key** and **API secret** and continue. Elementary validates the credentials and loads the destination groups available in your Fivetran account.
  </Step>
  <Step title="Select destination groups and connect">

>     Choose the destination groups you want to monitor, then save. Elementary connects to Fivetran and starts monitoring the selected groups automatically.

Author: ```suggestion
    Choose the destination groups you want to monitor, then save. Elementary connects to Fivetran and automatically starts syncing and monitoring the selected groups.
```

  </Step>
</Steps>

> ### Verify the connection

Author: Redundant

- Destination group ID

<Note>

>   At this time, Fivetran incidents cannot be filtered by connector — all connectors in the monitored destination groups are included.

Author: ```suggestion
  At this time, Fivetran incidents cannot be filtered by connector - all connectors in the monitored destination groups are included.
```

>   At this time, Fivetran incidents cannot be filtered by connector - all connectors in the monitored destination groups are included.

Author: Kind of duplication, we wrote above:
```
  **Connector filtering is not yet supported.** Incidents will be created for all connectors in the monitored destination groups. Filtering by specific connectors will be added in a future release.
```
